esm-samples/webpack/README.md

```diff
@@ -2,6 +2,10 @@
 
 This repo demonstrates how to use the [`@arcgis/core`](https://www.npmjs.com/package/@arcgis/core) ES modules with webpack.
 
+
+## Known Issues
+- ``webpack-dev-server`` had a [breaking change](https://github.com/webpack/webpack-dev-server/blob/master/CHANGELOG.md#-breaking-changes-4) in ``4.0.0`` which removed ``contentBase`` in favor of the ``static`` option. This sample has been changed accordingly.
```

Comment:
@benelan looks like one last small item, can you change the double backticks to single `backticks`?

Reply:
Good to go!

Reply:
Awesome, merging!